client/state/reducer.js

```diff
@@ -17,6 +21,7 @@ const reducers = {
 	currentUser,
 	dataRequests,
 	sites,
+	startupFlags,
```

Comment:
Ideally, we want to reduce this list to zero, not grow it further 😉

Reply:
Oh I thought this slice sort of makes sense to live in this file because it is meant as a global startup slice. Lazy registering using `registerReducer` felt unnecessary since this slice will anyways be loaded on every load. WDYT?

Reply:
Hey! Sorry, just came across this now.

Must it really be loaded on every boot? I was under the impression this was only needed in development builds.

Reply:
This has been removed.